rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverterSpec.groovy

```diff
@@ -34,8 +34,7 @@ class PackageNameConverterSpec extends Specification implements TestDefaults {
       null                                           | new PackageNameConverter.OsPackageName()
       ""                                             | new PackageNameConverter.OsPackageName()
       "nflx-djangobase-enhanced"                     | new PackageNameConverter.OsPackageName(name: "nflx-djangobase-enhanced")
-      "nflx-djangobase-enhanced_0.1"                 | new PackageNameConverter.OsPackageName(name: "nflx-djangobase-enhanced",
-                                                                                              version: "0.1")
+      "nflx-djangobase-enhanced_0.1"                 | new PackageNameConverter.OsPackageName(name: "nflx-djangobase-enhanced_0.1")
```

Comment:
As a result of this PR, having only version name is no longer enough to be parsed. But from what I can gather the BakeRequest value of the packageName would have been used in the previous versions. So in effect this should have the same result.